org-brain.el

```diff
@@ -25,15 +26,16 @@
 ;; link, just like the `org-mode' file-link: [[brain:index::*Programming]].
 
 ;; You can use `org-brain-visualize' to see the relationships between entries,
-;; quickly add parents/children to an entry, and open them for editing. In this
-;; view you may also have pinned entries, which will be shown at all times. To
-;; pin an entry, add #+BRAIN_PIN: on a line in the beginning of the entry file.
+;; quickly add parents/children/pins to an entry, and open them for editing. In
+;; this view you may also have pinned entries, which will be shown at all times.
+;; To pin an entry, add #+BRAIN_PIN: on a line in the beginning of the entry
+;; file.
 
 ;;; Code:
 
+(require 'cl)
```

Comment:
Is there a reason for using `cl` instead of `dash`? I know `cl` is included by default, so that is good. If `dash` is no longer required we should remove it from `Package-Requires` (line 10).